WF_Proton_Launcher/Main.cpp

```diff
@@ -130,6 +136,13 @@ int main(int argc, char** argv)
                         registry_option = arg;
 		if (arg == "-fullscreen")
 			fullscreen_option = " -fullscreen:1 ";
+		if (arg.substr(0, 10) == "-language:")
+		{
+			lang = arg.substr(10, 2);
+			auto found = std::find(std::begin(supported_languages), std::end(supported_languages), lang);
+			if (found != std::end(supported_languages))
+				language = arg;
```

Comment:
To repeat the complaint I had on the similar change for supporting selecting the registry;

With the code as it is here, if Warframe was to add localization for another language - Swedish for instance, then the launcher code would have to be updated to add `"se"` to the list of valid languages. If we just pass along any arbitrary code that the user specifies then we automatically become forward-compatible.  
So if the validation was skipped here, what would warframe do if it receives a launch argument to run with - for example - `-language:wf`? Would it give an error or would it silently ignore the invalid argument?

Reply:
In general, I'd much prefer if we leave validation up to the ones who control the exact list of valid inputs, that way we won't silently ignore valid input - or worse, give false positive errors on said valid input.